src/EA.Iws.Web/Areas/AdminExportAssessment/Views/Overview/Sections/_TransitStates.cshtml

```diff
@@ -18,6 +18,8 @@
                     <div class="column-half">
                         @transitState.EntryPoint.Name,
                         @transitState.Country.Name
+                        
+                        <div class="change-answer">@Html.ActionLink("Change", "TransitEntryPoint", "UpdateJourney", new { id = Model.NotificationId, transitStateId = transitState.Id }, null)</div>
```

Comment:
@SumeN - forgot to add logic here for hiding the link for non-admin users. Will be adding that now.

Reply:
Now fixed by this [commit](https://github.com/DEFRA/prsd-iws/pull/1005/commits/7e7f017a72489e3d5f2d0af680f726828fede152)